fontbe/src/features/marks.rs

```diff
@@ -1069,6 +1334,192 @@ mod tests {
         );
     }
 
+    // reduced from testdata/glyphs3/Oswald-AE-comb
+    //
+    // this ends up mostly being a test about how we generate mark filtering sets..
+    #[test]
+    fn oswald_ae_test_case() {
+        let mut input = MarksInput::default();
+        let out = input
+            .add_glyph("A", None, |anchors| {
+                anchors
+                    .add("bottom", [(234, 0)])
+                    .add("ogonek", [(411, 0)])
+                    .add("top", [(234, 810)]);
+            })
+            .add_glyph("E", None, |anchors| {
+                anchors
+                    .add("topleft", [(20, 810)])
+                    .add("bottom", [(216, 0)])
+                    .add("ogonek", [(314, 0)])
+                    .add("top", [(217, 810)]);
+            })
+            .add_glyph("acutecomb", None, |anchors| {
+                anchors.add("top", [(0, 810)]).add("_top", [(0, 578)]);
+            })
+            .add_glyph("brevecomb", None, |anchors| {
+                anchors.add("top", [(0, 810)]).add("_top", [(0, 578)]);
+            })
+            .add_glyph("tildecomb", None, |anchors| {
+                anchors.add("top", [(0, 742)]).add("_top", [(0, 578)]);
+            })
+            .add_glyph("macroncomb", None, |anchors| {
+                anchors.add("top", [(0, 810)]).add("_top", [(0, 578)]);
+            })
+            .add_glyph("breveinvertedcomb", None, |anchors| {
+                anchors
+                    .add("top", [(0, 810)])
+                    .add("top.a", [(0, 810)])
+                    .add("_top.a", [(0, 578)]);
+            })
+            .get_normalized_output();
+
+        assert_eq_ignoring_ws!(
+            out,
+            // this gets an abvm feature because we don't specify any languagesystems
```

Comment:
I don't understand this, why would this get "an abvm feature because we don't specify any languagesystems"?

Reply:
oh I see.. because this font doesn't have explicit languagesystems declared in the FEA, which would filter the scripts actually used, and because acutecomb's script extensions include 'Tale' (Tai Le) script and macroncomb's script extensions includ 'Tfng' (Tifinagh) script, they get slotted in the 'abvm' feature... Very edge case, tbh. More confusing than it is helpful. But I suppose this matches ufo2ft so, ok.

Reply:
I was also confused by this, but it matches fonttools. Basically because of this: https://github.com/googlefonts/ufo2ft/blob/16ed156bd6a8b9bc035d0aa8045a1271ef79a52e/Lib/ufo2ft/featureWriters/markFeatureWriter.py#L1122

if there are no feaScripts declared, we don't modify the `scriptsUsingAbvm` value, and then several of these combining accents are used in abvm scripts.